src/content/docs/en/pages/main-menu/reference/observe/real-time-events/real-time-events.mdx

```diff
@@ -41,261 +41,271 @@ You can use Real-Time Events to:
-| $client_id | Unique Azion customer identifier. Example: `4529r` |
-| $geoloc_country_name | Remote client’s country detected via IP address geolocation. Example: `United States`, `Russian Federation` |
-| $host | Host information sent on the request line. Stores: host name from the request line, or host name from the `Host` request header, or the server name matching a request. Example: `website.com.br` |
-| $remote_address | IP address of the origin that generated the request. Example: `54.233.153.15` |
-| $server_protocol | The connection established protocol. Example: `HTTP/1.1`, `HTTP/2.0`, `HTTP/3.0` |
-| $time | Request date and time. Example: `31 October, 2023, 21:10:55` |
-| $total_blocked | Informs the total number of requests blocked by WAF. Example: `2` |
-| $total_processed | Informs the total number of requests processed by WAF. Example: `1` |
-| $waf_action | Reports WAF’s action regarding the action. Can be: `$BLOCK`, `$PASS`, `$LEARNING_BLOCK`, or `$LEARNING_PASS`. |
-| $waf_args | The request arguments. Example: `quantidade_periodos=10` |
-| $waf_learning | Informs if WAF is in Learning mode. Can be `0` or `1` |
-| $waf_match | List of infractions found in the end user’s request. It’s formed by key-value elements: the *key* refers to the type of violation detected; the *value* shows the string that generated the infraction. Example: `0:1311:BODY:ctl00_cph_jp1_dados_container_clientstate`. Find out more on [WAF Allowed Rules](/en/documentation/products/secure/edge-firewall/web-application-firewall/#allowed-rules) |
-| $waf_score | Reports the score that'll be increased in case of a match with the rules set for the WAF. Example: `0:$SQL:2` |
-| $waf_server | Hostname used in the WAF request. Example: `api-login.azion.com.br` |
-| $waf_uri | URI used in the WAF request. Example: `/access/v2/after-login` |
+| Bytes Sent | Number of bytes sent to a client. This field is the result of a sum. Example: `191` |
+| Cache Key | The stored object cache identification key for the content requested by a client. Example: `/index.html` |
+| Cache TTL | Time, in seconds, the cached object is considered valid (not expired). After the time expiration, when a new request occurs, L2 Caching queries the data on the origin (upstream). Example: `31536000` |
+| Configuration ID | Unique Azion configuration identifier set on virtual host configuration file. Example: `1595368520` |
+| Host | Host information sent on the request line. Stores: host name from the request line, or host name from the `Host` request header field, or the server name matching a request. Example: `g1sdetynmxe0ao.map.azionedge.net` |
+| Proxy Host | Hostname being proxied. Example: `storage.googleapis.com:443` |
+| Proxy Status | HTTP error status code or origin when no response is obtained from the upstream. Example: `520`. **In case of cache, the response is `-`.** |
```

Comment:
```suggestion
| Proxy Status | HTTP error status code. May return an origin address when no response is obtained from the upstream. Example: `520`. **In case of cache, the response is `-`.** |
```

🤔